app/services/deployment/processes/create_attached_project_instance.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Deployment
+  module Processes
+    class CreateAttachedProjectInstance
+      def initialize(project, current_user)
+        @project = project
+        @current_user = current_user
+      end
+
+      def call(project_instance_name:, branches: {}, pull_request_number: nil)
```

Comment:
Since it's an attached ProjectInstance, it has to have a pull request number. So please remove the default `nil` value